makefile

```diff
@@ -1,26 +1,27 @@
 # Helper function to perform recursive search in a directory for file types.
 rwildcard = $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2) $(filter $(subst *,%,$2),$d))
 
-SRC_DIR 	:= src
-BIN_DIR 	:= bin
-OBJECT_DIR 	:= obj
+SRC_DIR 		:= src
+BIN_DIR 		:= bin
+OBJECT_DIR 		:= obj
```

Comment:
Weird tabbing here. I have to check my git settings.

Reply:
Oh turns out my github appearance settings had tabs set to 8. Didn't know you could do that.